src/status_im/navigation/transitions.cljs

```diff
@@ -44,15 +44,15 @@
 (def stack-slide-transition
   {:push {:content {:translationX {:from     (:width (rn/get-window))
                                    :to       0
-                                   :duration 200}}}
+                                   :duration 250}}}
```

Comment:
small change related to https://discord.com/channels/624634427930312714/948957499829149747/1252555238414614589
(change is too small and doesn't seem worth different issue or PR)